src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -84,10 +85,10 @@ public void storeListing(Listing listing) {
    * @return a list of listing posted by the user, or empty list if user has never posted a
    *     listing. List is sorted by time descending.
    */
-  public List<Listing> getListings(String user) {
+  public List<Listing> getListings(String key, String searchIndex) {
```

Comment:
made the get Listings function more modular by allowing dynamic searching of a datastore entity